cmd/utils/utils.go

```diff
@@ -20,15 +20,15 @@ const (
 
 	DefaultCoreOperatorDockerImage = "ghcr.io/calyptia/core-operator"
 	// DefaultCoreOperatorDockerImageTag not manually modified, CI should switch this version on every new release.
-	DefaultCoreOperatorDockerImageTag = "1.0.9"
+	DefaultCoreOperatorDockerImageTag = "v1.0.9"
 
 	DefaultCoreOperatorToCloudDockerImage = "ghcr.io/calyptia/core-operator/sync-to-cloud"
 	// DefaultCoreOperatorToCloudDockerImageTag not manually modified, CI should switch this version on every new release.
-	DefaultCoreOperatorToCloudDockerImageTag = "1.0.9"
+	DefaultCoreOperatorToCloudDockerImageTag = "v1.0.9"
 
 	DefaultCoreOperatorFromCloudDockerImage = "ghcr.io/calyptia/core-operator/sync-from-cloud"
 	// DefaultCoreOperatorFromCloudDockerImageTag not manually modified, CI should switch this version on every new release.
-	DefaultCoreOperatorFromCloudDockerImageTag = "1.0.9"
+	DefaultCoreOperatorFromCloudDockerImageTag = "v1.0.9"
```

Comment:
this is being done by the ci so on next ci release it will be changed

Reply:
I had to change it so I can test it.
Next version is going to be v1.0.10 anyways

Reply:
I'm thinking that these values should be baked in as LDFLAGS. 

@patrick-stephens do you have any prefernces?

Reply:
LDFLAGS would be nicer I agree but it's probably a larger change